src/main/java/net/atos/entng/calendar/services/impl/EventServiceMongoImpl.java

```diff
@@ -207,6 +208,9 @@ public void update(String eventId, JsonObject body, UserInfos user, Handler<Eith
         // Modifier
         MongoUpdateBuilder modifier = new MongoUpdateBuilder();
         for (String attr : body.fieldNames()) {
+            if (attr.equals(Field.ATTACHMENTS)) {
```

Comment:
filter unwanted file to allow to save events with attachments (quick fix with ticket programmed for later)